README.md

```diff
@@ -33,6 +33,33 @@ tabrmd on the system bus as this is the most common configuration.
 
 Check out the man page TCTI-TABRMD(7) and TSS2_TCTI_TABRMD_INIT(3).
 
+## tpm2-abrmd vs in-kernel RM
+The current implementations are mostly equivalent with a few differences.
+Both provide isolation between objects & sessions created by different connections which is the core functionality required by applications.
+The reason we have both is that the in-kernel RM was only added very recently and we have TPM2 users in environments with kernels going back to the 3.x series.
+So the user space RM will be around at least till everyone is using the kernel RM.
+
+For the short term we're recommending that developers stick to using the tabrmd as the default to get the most stable / widest possible support.
+If you structure your code properly you'll be able to switch in / out TCTI modules with relative ease and migrating to the in-kernel RM should be pretty painless.
+Eventually, all of the required features will end up in the kernel RM and it will become the default.
+
+How we get to the ideal future of a single RM in the kernel: our current plan is to prototype various features in user space as a way to get them tested / validated.
+There's a lot of stuff in the related TCG spec that we haven't yet implemented and we all agree that trying to put features into the kernel before we:
+1. understand how they work
+2. how they're going to be used by applications
+3. agree we want the feature at all
+
+is generally a bad idea.
```

Comment:
I would move this last sentence above the three points mentioned, i.e: "we all agree that's generally a bad idea trying to put features into the kernel before we"

Otherwise if people are not reading carefully, they may have to go back and re-read the sentence before the points to remember what was a bad idea.

Reply:
+1